snowexsql/api.py

```diff
@@ -198,9 +231,13 @@ def all_observers(self):
         Return all distinct observers in the data
         """
         with db_session(self.DB_NAME) as (session, engine):
-            qry = session.query(self.MODEL.observers).distinct()
+            qry = session.query(Observer).join(
+                self.LINK_TABLE_MODEL,
+                Observer.id == self.LINK_TABLE_MODEL.observer_id
+            ).distinct()
```

Comment:
Do we need a join here?
Same as with the sites - My brain tells me we can just do a
```sql
Select name from observers
```
and be done

Reply:
Haha you are correct